### DISCUSSION TITLE
Orient the floating window for vertical screen

### DISCUSSION THREAD

**Discussion by misraelson** [junegunn/fzf#2807] [General]
I have my coding screen vertical. I was wondering if it was possible to orient the floating window to reflect this.
For example have the preview window below the search results sectio
![Screen Shot 2022-04-28 at 8 58 09 PM](https://user-images.githubusercontent.com/17835964/165883534-ed33f9d2-d98f-450c-90f0-8e51812b30fd.png)
n

**Comment by tg0h** [comment]
![Screenshot 2022-05-03 at 10 36 27 - iTerm2](https://user-images.githubusercontent.com/7954426/166397088-62fa38c5-50e5-4520-a11a-dcd86ed77164.gif)

sample command: `ls | fzf --preview-window=down,80% --preview='cat {}' --bind 'ctrl-t:change-preview-window(right,80|hidden|down,80%)`
- show the preview window at the bottom by default
- bind ctrl t to toggle the preview window position

have a look at the 
- preview-window https://github.com/junegunn/fzf#preview-window
- change-preview-window command  https://www.mankier.com/1/fzf#Key/Event_Bindings-Change_Preview_Window_Attributes
